pkg/platform/aarch64.go

```diff
@@ -28,21 +28,21 @@ func (p *Aarch64) GetPackages() []string {
 	return packages
 }
 
-type Aarch64_IoT struct {
+type Aarch64_Fedora struct {
```

Comment:
Can you elaborate on the reason to rename this platform? I always thought that the platform is distro-agnostic until you instantiate it set custom values...

Reply:
I think the intention was to remove reference to IoT as its applicable to all of Fedora. Perhaps `Aarch64_Sbc` or `Aarch64_Sr`  to be more generic?

Reply:
Personally, I would even merge it all into the `Aarch64` platform and have just one. I don't like distro-specific code leaking outside of the `distro` directory. @achilleas-k do you have any opinions on this?

Reply:
I agree with both things you said but I'm okay with having it messy like this for now and cleaning up after.  It was already a bit weird that we had this `IoT` variant in the platform package.  Like you said, there's no reason we can't have the extra `BootFiles` in the base struct and leave it empty for the ones that don't need it.

But we can merge them in a follow-up and keep the rename.

Reply:
OK, I'm fine with a follow-up. Nevertheless, there is still the question of the missing offset for RHEL-8 images...